system-addon/content-src/components/Sections/Sections.jsx

```diff
@@ -1,25 +1,56 @@
 const React = require("react");
 const {connect} = require("react-redux");
-const {FormattedMessage} = require("react-intl");
+const {injectIntl, FormattedMessage} = require("react-intl");
 const Card = require("content-src/components/Card/Card");
 const Topics = require("content-src/components/Topics/Topics");
 
 class Section extends React.Component {
+  constructor(props) {
+    super(props);
+    this.onMouseOverSectionInfoOption =
+      this.onMouseOverSectionInfoOption.bind(this);
+    this.onMouseOutSectionInfoOption =
+      this.onMouseOutSectionInfoOption.bind(this);
+    this.state = {isHovering: false};
+  }
+
+  onMouseOverSectionInfoOption() {
+    this.setState({isHovering: true});
+  }
+
+  onMouseOutSectionInfoOption() {
+    this.setState({isHovering: false});
+  }
+
   render() {
-    const {id, eventSource, title, icon, rows, infoOption, emptyState, dispatch, maxCards, contextMenuOptions} = this.props;
+    const {id, eventSource, title, icon, rows, infoOption, emptyState, dispatch, maxCards, contextMenuOptions, intl} = this.props;
     const initialized = rows && rows.length > 0;
     const shouldShowTopics = (id === "TopStories" && this.props.topics && this.props.read_more_endpoint);
+
+    const infoOptionIconA11yAttrs = {
+      "aria-haspopup": "true",
+      "aria-controls": "info-option",
```

Comment:
yeah, there appears to be some workarounds for this in react, i'll file a spin-off bug to deal with that by 57.